### PR TITLE
Bump image version

### PR DESCRIPTION
**Pull request by brendarearden**
**Problem**
- Need to update to include grafana dashboard changes
- Kiali dependencies is still not very clear and inconsistency in capitalization 

**Solution**
- Need to bump installer version to add the grafana dashboard changes
- Updated readme to make some clarifications for the kiali dependencies, and added consistent capitalization. 

**Issues**
https://github.com/rancher/rancher/issues/28468
https://github.com/rancher/rancher/issues/29418

**Additional Info**
Since rancher-istio 1.7.3 chart is not in production, not bumping version - will just overwrite existing dev chart

### CODE REVIEW

**Review by nickgerace — DISMISSED** [review]

**Review by paynejacob — APPROVED** [review]

**Review by paynejacob — DISMISSED** [review]

**Comment by brendarearden** [comment]
removed some wording from the >note section in the readme and app-readme

**Review by paynejacob — APPROVED** [review]

File: packages/rancher-istio/charts/values.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ forceInstall: false
 
 installer:
   repository: rancher/istio-installer
-  tag: 1.7.3-rancher1
+  tag: 1.7.3-rancher2
```
Comment (cbron) [inline packages/rancher-istio/charts/values.yaml:9]: I thought we added a grafana flag for on or off, but I guess not. It just runs if its able. We might want to add that later.

**Review by cbron — APPROVED** [review]
Already new istio version in charts so no need to bump app version here

**Review by nickgerace — APPROVED** [review]

File: packages/rancher-istio/charts/values.yaml (hunk 1 above)
Comment (brendarearden) [reply packages/rancher-istio/charts/values.yaml:9]: Yeah, it seemed like grafana won't install unless rancher-monitoring is installed and you would set grafana enabled there....

**Pull request merged**